packages/pfff/pfff.0.37.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "yoann.padioleau@gmail.com"
+authors: [ "Yoann Padioleau <yoann.padioleau@gmail.com>" ]
+homepage: "https://github.com/facebook/pfff/wiki/Main"
+patches: [
+]
+build: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "depend"]
+  [make]
+  [make "opt"]
+]
+install: [
+  [make "install-libs"]
+]
+remove: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "uninstall-libs"]
+]
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "ocamlfind"
+  "camlp4"
+  "num"
+  "ocamlgraph"
+  "json-wheel"
+]
+bug-reports: "https://github.com/aryx/pfff/issues"
+dev-repo: "git+https://github.com/aryx/pfff.git"
+synopsis:
+  "Tools and APIs for program analysis, code visualization, refactoring"
+description: """
+pfff is a set of tools and APIs to perform some static analysis,
+dynamic analysis, code visualizations, code navigations, or
+style-preserving source-to-source transformations such as refactorings
+on source code. For now the effort is focused on PHP but there is also
+good support for C, C++, Java, HTML, Javascript, Css, and preliminary
+support for Erlang, Python, C#, Lisp, Scheme, Haskell, OPA, Sql, and
+even TeX. There is also very good support for OCaml and noweb
+(literate programming) so that pfff can be used on the code of pfff
+itself.
+
+For more information see https://github.com/facebook/pfff/wiki/Main"""
+extra-files: [
+]
+url {
+  src: "https://github.com/aryx/pfff/archive/0.37.1.tar.gz"
```

Comment:
Could you clarify the status of this? I see facebook stopped working on this a few years ago. Is it the new official repository or just your fork?

Reply:
I am the main author of pfff and I left facebook in 2014. It is not maintained anymore by facebook
so I've forked it. I did not migrate yet the wiki to my fork, hence the mix of URLs.